kitsune/settings.py

```diff
@@ -911,6 +911,7 @@ def JINJA_CONFIG():
 GA_ACCOUNT = config('GA_ACCOUNT', 'something@developer.gserviceaccount.com')  # Google API Service Account email address
 GA_PROFILE_ID = config('GA_PROFILE_ID', default='12345678')  # Google Analytics profile id for SUMO prod
```

Comment:
This is used in [`googleanalytics.py`](https://github.com/mozilla/kitsune/blob/master/kitsune/sumo/googleanalytics.py) which pulls GA stats into Kitsune. 

Is GTM compatible with this? In other other words will it populate the same tables / properties (or how they are called in GA)?

Reply:
Just read your comment in #3027 . Will continue the discussion there.